### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Chkhikvadze**

### CODE REVIEW

File: datura/tools/twitter/twitter_advanced_search_tool.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        description="The search query for tweets.",
+    )
+
+
+class TwitterAdvancedSearchTool(BaseTool):
+    """Tool that gets tweets from Twitter."""
+
+    name = "Twitter Advanced Search"
+
+    slug = "get_tweets"
```
Comment (Nadejde) [inline datura/tools/twitter/twitter_advanced_search_tool.py:25]: The slug and tool ID is the same as on the twitter_search_tool. Is this just not finished yet? Both tools are exposed throught the toolkit.

File: datura/tools/twitter/twitter_advanced_search_tool.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+        client = TwitterScraperActor()
+
+        tweets = await client.get_tweets_advanced(
+            urls=[
+
+            ],
+            start=start_date,
+            end=end_date,
+            searchTerms=["Crypto"],
```
Comment (Nadejde) [inline datura/tools/twitter/twitter_advanced_search_tool.py:58]: Hardcoded search terms.

File: datura/tools/twitter/twitter_toolkit.py [hunk 3]
```diff
@@ -12,9 +13,10 @@ class TwitterToolkit(BaseToolkit, ABC):
     toolkit_id = "0e0ae6fb-0f1c-4d00-bc84-1feb2a6824c6"
 
     def get_tools(self) -> List[BaseTool]:
-        return [TwitterSearchTool()]
+        return [TwitterSearchTool(), TwitterAdvancedSearchTool()]
```
Comment (Nadejde) [inline datura/tools/twitter/twitter_toolkit.py:16]: these tools have the same slug and uid

Comment (Chkhikvadze) [reply datura/tools/twitter/twitter_toolkit.py:16]: Is not finished yet, just run in development mode

File: datura/tools/twitter/twitter_advanced_search_tool.py (hunk 2 above)
Comment (Chkhikvadze) [reply datura/tools/twitter/twitter_advanced_search_tool.py:58]: Is not finished yet, just run in development mode

**Pull request merged**